5_data_munge.yml

```diff
@@ -78,8 +84,9 @@ targets:
       nwis_dv_sites_ind = '2_nwis_pull/inout/nwis_dv_inventory.feather.ind',
       nwis_uv_sites_ind = '2_nwis_pull/inout/nwis_uv_inventory_reduced.feather.ind',
       ecosheds_sites_ind = '4_other_sources/out/ecosheds_sites.rds.ind',
-      wqp_sites_ind = '1_wqp_pull/inout/wqp_inventory.feather.ind')
-      
+      wqp_sites_ind = '1_wqp_pull/inout/wqp_inventory.feather.ind',
+      norwest_sites_ind = '4_other_sources/out/norwest_raw_files.yml')
```

Comment:
if you make the suggested additions to 4_other_sources.yml , you can then specify the data and site files independently here